### ISSUE TITLE
In search of a new maintainer.

### ISSUE THREAD

**Issue by adunkman** [adunkman/connect-assets#357]
Hi!

I don’t really care about this anymore, which is probably frustrating for all of you that are using it.

Would you be interested in taking over the project?

**Comment by js-kyle** [comment]
Hi @adunkman 

I'd like to put myself forward for this. I've been using this project myself for a while, and have submitted  a PR for some improvements in the past. Let me know what you think.

**Comment by adunkman** [comment]
Hey @js-kyle! That would be lovely! 

Let me know your NPM username, and I’ll begin the process of transferring the package over to you and transfer the repository to your GitHub.

**Comment by js-kyle** [comment]
@adunkman Cool! It matches this one, `js-kyle`

**Comment by js-kyle** [comment]
@adunkman Do you need anything from me?

**Comment by adunkman** [comment]
@js-kyle Nope, sorry for the delay! Added you as an NPM owner just now, and requested a transfer of the repository to you on GitHub.
